spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+RSpec.describe Airbrake::Shoryuken::ErrorHandler do
+  let(:error) { AirbrakeTestError.new('shoryuken error') }
+  let(:body) { { message: 'message' } }
+  let(:notice) { double 'notice' }
+  let(:worker) { FooWorker.new }
+  let(:endpoint) do
+    'https://airbrake.io/api/v3/projects/113743/notices?key=fd04e13d806a90f96614ad8e529b2822'
+  end
+
+  class FooWorker; end
+
+  def wait_for_a_request_with_body(body)
+    wait_for(a_request(:post, endpoint).with(body: body)).to have_been_made.once
+  end
```

Comment:
Yeah, I agree that this doesn't look good, however we use this method in many spec files, so this change is rather a job for a different PR because it's quite involving.